kyo-core/shared/src/main/scala/kyo/Resource.scala

```diff
@@ -124,7 +90,7 @@ object Resource:
       *   The acquired Closeable resource wrapped in Resource, IO, and S effects.
       */
     def acquire[A <: java.io.Closeable, S](resource: A < S)(using Frame): A < (Resource & IO & S) =
-        acquireRelease(resource)(r => IO(r.close()))
+        acquireRelease(resource)(_.close())
```

Comment:
The `IO` suspension here wasn't necessary since it's a by-name param that is already deferred.